src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -63,6 +64,51 @@ and positive-definite scale matrix Sigma; may only be used in
 transformed data and generated quantities blocks
 `r since("2.0")`
 
+## Wishart distribution, Cholesky Parameterization (#wishart-cholesky-distribution)
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1,\infty)$, and $L_S \in
+\mathbb{R}^{K \times K}$ is lower triangular and such that $L_SL_S^{\top}$
+is positive definite,then for
+$L_W \in \mathbb{R}^{K \times K}$ is lower triangular and such that $L_WL_W^{\top}$
+is positive definite, \[
+\text{WishartCholesky}(L_W|\nu,L_S) =
+\text{Wishart}(W|\mu,LL^{\top}). \] If $L$ such that $L \in {L_S, L_W}$  is lower triangular and
+$L^L^{top}$ is a $K \times K$ positive definite matrix, then $L_{k,k}$
+must be strictly positive for $k \in 1{:}K$. If an $L$ is provided
+that is not the Cholesky factor of a positive-definite matrix, the
+probability functions will raise errors.
+
+### Stan functions
+
+<!-- real; wishart_cholesky_lpdf; (matrix L_W | real nu, matrix L_S); -->
+\index{{\tt \bfseries wishart\_cholesky\_lpdf }!{\tt (matrix L_W \textbar\ real nu, matrix L_S): real}|hyperpage}
+
+`real` **`wishart_cholesky_lpdf`**`(matrix L_W | real nu, matrix L_S)`<br>\newline
+The log of the Wishart density for lower-triangular Cholesky factor of L_W
+given degrees of freedom nu and lower-triangular Cholesky factor of the
+scale matrix L_S
```

Comment:
`L_W` and `L_S` should be set with back ticks to indicate they are arguments.  This goes for all of these functions.

Reply:
I added the back ticks but I don't see back ticks for these on the other distributions. For eg,
```
`real` **`inv_wishart_lupdf`**`(matrix W | real nu, matrix Sigma)`<br>\newline
The log of the inverse Wishart density for symmetric and
positive-definite matrix W given degrees of freedom nu and symmetric
and positive-definite scale matrix Sigma dropping constant additive terms
`r since("2.25")`
```

Reply:
You're right.  There is no consistency in our doc.  Some functions do what I consider the right thing and format the arguments and others don't.  So do whatever you want.

Reply:
Ok, maybe I'll fix the rest of the docs if I feel ambitious. This should be good to merge.